config/routes.php

```diff
@@ -52,7 +52,7 @@
      * its action called 'display', and we pass a param to select the view file
      * to use (in this case, templates/Pages/home.php)...
      */
-    $builder->connect('/', ['controller' => 'Pages', 'action' => 'display', 'home']);
+    $builder->connect('/', ['controller' => 'Posts', 'action' => 'index']);
```

Comment:
Hmm.. I would rather keep "/" mapped to the default homepage since it shows basics checks. The README already mentions that one needs to access `/posts` to view the Crud page.

Reply:
What if I add links to posts & comments on the home page? Maybe I'm super lazy, but I find it annoying to have to manually type in links I may not be familiar with in the address bar rather than having them provided for easy click.

Reply:
> What if I add links to posts & comments on the home page? 

Sounds great 👍.